src/lib/list/list.spec.ts

```diff
@@ -156,6 +156,49 @@ describe('MdList', () => {
 
     items.forEach(item => expect(item.isRippleEnabled()).toBe(false));
   });
+
+  it('should allow enabling ripples for a normal list', () => {
+    const fixture = TestBed.createComponent(ListWithOneAnchorItem);
+    const list = fixture.debugElement.query(By.directive(MdList)).componentInstance;
+
+    fixture.detectChanges();
+
+    const items = fixture.componentInstance.listItems;
+
+    expect(items.length).toBeGreaterThan(0);
+    expect(list.disableRipple).toBe(true, 'Ripples should be disabled for normal lists.');
+
+    // Ripples should be disabled by default, and can be enabled with a binding.
+    items.forEach(item => expect(item.isRippleEnabled()).toBe(false));
```

Comment:
Here and on line 177, instead of doing multiple `it` calls inside a loop, you could use `Array.every`. The difference is that the log will be much shorter if it fails:

```ts
expect(items.every(item => item.isRippleEnabled())).toBe(false);
```